REQUIRE

```diff
@@ -7,3 +7,4 @@ Reexport
 FixedSizeArrays
 Measures
 Showoff
+StatsBase
```

Comment:
Good point - I have been meaning to ask the StatsBase people when they'll make the release, so we can time the Plots release. I'll do that straigth away.